### PR TITLE
Show all action templates in categories

### PR DESCRIPTION
**Pull request by moritzraho**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: generators/add-action/index.js [hunk 1]
```diff
@@ -65,33 +69,104 @@ class AddActions extends Generator {
...
+ **/
+function getPromptChoices (adobeServicesOption, supportedAdobeServicesOption) {
+  // helpers
+
+  const cleanInputServices = (option) => option
+    .split(',')
+    .map(x => x.trim())
+    .filter(s => s)
+
+  const toChoices = (sdkCodeSet, checked = false) => [...sdkCodeSet]
```
Comment (purplecabbage) [inline generators/add-action/index.js:117]: This requires a description .. what is happening here?
Comments are free!

File: lib/constants.js [hunk 2]
```diff
@@ -1,4 +1,6 @@
 module.exports = {
+  promptMaxChoices: 12,
```
Comment (purplecabbage) [inline lib/constants.js:2]: Why a max now? How will we see more than 12?
Personally, I would delay addition of this until a) it is a problem, or b) we can handle more than 12 and not just ignore them

Comment (moritzraho) [reply lib/constants.js]: the default max is lower than the number of inputs, 12 was an arbitrary number (some number to delay the problem 😉 )
I've changed this to the size of inputs

Comment (moritzraho) [reply lib/constants.js]: also the max here is not a hardlimit, inquirer will still let you navigate through all choices even if you have 100

**Comment by moritzraho** [comment]
ok now the PR is ready for review 🙂

**Pull request merged**